addons/project/models/project_update.py

```diff
@@ -46,7 +47,8 @@ def default_get(self, fields):
         ('on_track', 'On Track'),
         ('at_risk', 'At Risk'),
         ('off_track', 'Off Track'),
-        ('on_hold', 'On Hold')
+        ('on_hold', 'On Hold'),
+        ('done', 'Done')
```

Comment:
```suggestion
        ('done', 'Done'),
```
So that next time we need to add a value, there won't be a useless diff on this line :innocent: